src/custom-element.js

```diff
@@ -1488,21 +1489,21 @@ function createBaseCustomElementClass(win) {
      * Turns the loading indicator on or off.
      * @param {boolean} state
      * @param {boolean=} opt_cleanup
-     * @private @final @this {!Element}
+     * @public @final @this {!Element}
      */
-    toggleLoading_(state, opt_cleanup) {
+    toggleLoading(state, opt_cleanup) {
       assertNotTemplate(this);
-      if (state &&
-          (this.layoutCount_ > 0 ||
-              this.signals_.get(CommonSignals.RENDER_START))) {
-        // Loading has already been canceled. Ignore.
-        return;
-      }
       this.loadingState_ = state;
       if (!state && !this.loadingContainer_) {
         return;
       }
 
+      if (state && !this.implementation_.doesReuseLoadingIndicator() &&
```

Comment:
any reason this block moved down?

Reply:
Not that I remember. I'm not sure how this got moved. I moved it back now.